docs/installation_guides/installation_general_info.md

```diff
@@ -1,4 +1,7 @@
-# Installing RHMI Using OCM
+# Overview
```

Comment:
I know you only added the Overview section. But I am question what is the need for this document. The getting started guide list cluster as a prerequisite. That requisite does need update as it lists 4.6 as the version but that is a different matter.

It would be of my opinion that this document can be completely removed.

Reply:
Agree, removed